src/view/draw.ts

```diff
@@ -23,6 +41,22 @@ const G_view_clearScreen = () => {
   );
 };
 
+const G_view_drawText = (
+  text: string,
+  x: number,
+  y: number,
+  color?: string,
+  scale?: number,
+  ctx?: CanvasRenderingContext2D
+) => {
+  scale = scale || 2;
+  color = color || 'black';
+  ctx = ctx || G_model_getCtx();
+  ctx.font = '14px serif';
+  ctx.fillStyle = color;
+  ctx.fillText(`${text}`, x, y);
```

Comment:
x, y should be multiplied by scale like it is for other fundamental drawing functions